File: Examples/CSharp/Files/Handling/SaveInPdfFormat.cs
using System.IO;
using System.Web;
using Aspose.Cells;

namespace Aspose.Cells.Examples.CSharp.Files.Handling
{
    public class SaveInPdfFormat
    {
        public static void Run()
        {
            // ExStart:1
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
            HttpResponse Respose = null;
            // Creating a Workbook object
            Workbook workbook = new Workbook();
              // Save in Pdf format
            workbook.Save(Respose, dataDir + "output.pdf", ContentDisposition.Attachment, new PdfSaveOptions());
            Respose.End();
            // ExEnd:1
        }
    }
}

File: Examples/VisualBasic/Files/Handling/SaveInPdfFormat.vb
Imports System.IO
Imports System.Web
Imports Aspose.Cells

Namespace Files.Handling
    Public Class SaveInPdfFormat
        Public Shared Sub Run()
            ' ExStart:1
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType)
            Dim Respose As HttpResponse = Nothing
            ' Creating a Workbook object
            Dim workbook As New Workbook()
            If Respose IsNot Nothing Then
                ' Save in Pdf format
                workbook.Save(Respose, dataDir + "output.pdf", ContentDisposition.Attachment, New PdfSaveOptions())
                Respose.[End]()
            End If
            ' ExEnd:1
        End Sub

    End Class
End Namespace
